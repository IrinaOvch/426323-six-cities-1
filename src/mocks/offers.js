export default [
  {
    id: 1,
    city: `Amsterdam`,
    title: `Beautiful & luxurious studio at great location`,
    img: `img/apartment-01.jpg`,
    price: 120,
    isPremium: true,
    rating: 4,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [52.3909553943508, 4.85309666406198],
  },
  {
    id: 2,
    city: `Amsterdam`,
    title: `Wood and stone place`,
    img: `img/apartment-02.jpg`,
    price: 80,
    isPremium: false,
    rating: 2,
    type: `Private room`,
    isInBookmarks: true,
    coordinates: [52.369553943508, 4.85309666406198],
  },
  {
    id: 3,
    city: `Amsterdam`,
    title: `Canal View Prinsengracht`,
    img: `img/apartment-03.jpg`,
    price: 132,
    isPremium: false,
    rating: 3,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [52.3909553943508, 4.929309666406198],
  },
  {
    id: 4,
    city: `Amsterdam`,
    title: `Nice, cozy, warm big bed apartment`,
    img: `img/apartment-02.jpg`,
    price: 180,
    isPremium: true,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [52.3809553943508, 4.939309666406198],
  },
  {
    id: 5,
    city: `Paris`,
    title: `Nice, cozy, warm big bed apartment`,
    img: `img/apartment-02.jpg`,
    price: 180,
    isPremium: true,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [48.8589507, 2.2770204],
  },
  {
    id: 6,
    city: `Paris`,
    title: `Nice, cozy, warm big bed apartment`,
    img: `img/apartment-02.jpg`,
    price: 180,
    isPremium: true,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [48.8589507, 2.2770204],
  },
  {
    id: 7,
    city: `Paris`,
    title: `Nice, cozy, warm big bed apartment`,
    img: `img/apartment-01.jpg`,
    price: 180,
    isPremium: true,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [48.8589507, 2.2770204],
  },
  {
    id: 8,
    city: `Cologne`,
    title: `Nice, cozy, warm big bed apartment`,
    img: `img/apartment-03.jpg`,
    price: 180,
    isPremium: false,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [50.9616947, 6.9295803],
  },
  {
    id: 9,
    city: `Cologne`,
    title: `Nice, cozy, warm big bed apartment`,
    img: `img/apartment-01.jpg`,
    price: 180,
    isPremium: true,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [50.9616947, 6.9295803],
  },
  {
    id: 10,
    city: `Brussels`,
    title: `Nice, cozy, warm big bed apartment`,
    img: `img/apartment-01.jpg`,
    price: 180,
    isPremium: true,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [50.8797438, 4.3149145],
  },
  {
    id: 11,
    city: `Brussels`,
    title: `Nice, cozy, warm big bed apartment`,
    img: `img/apartment-03.jpg`,
    price: 180,
    isPremium: false,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [50.796789, 4.4046897],
  },
  {
    id: 12,
    city: `Hamburg`,
    title: `Beautiful & luxurious studio at great location`,
    img: `img/apartment-01.jpg`,
    price: 180,
    isPremium: true,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [53.5650616, 9.9568253],
  },
  {
    id: 13,
    city: `Hamburg`,
    title: `Nice, cozy, warm big bed apartment`,
    img: `img/apartment-03.jpg`,
    price: 180,
    isPremium: false,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [53.5327143, 10.0225135],
  },
  {
    id: 14,
    city: `Dusseldorf`,
    title: `Beautiful & luxurious studio at great location`,
    img: `img/apartment-02.jpg`,
    price: 180,
    isPremium: true,
    rating: 5,
    type: `Apartment`,
    isInBookmarks: false,
    coordinates: [51.2385306, 6.8055928],
  },

];
