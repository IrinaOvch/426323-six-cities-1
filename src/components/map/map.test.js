import React from 'react';
import renderer from 'react-test-renderer';

import Map from './map.jsx'
import offers from '../../mocks/offers.js';
import mapData from '../../mocks/map-data.js';
import leaflet from '../../mocks/leaflet.js';

it(`should render map correctly`, () => {
  const map = renderer.create(
    <Map
      offers={offers}
      mapData={mapData}
      leaflet={leaflet}
    />
  ).toJSON();

  expect(map).toMatchSnapshot();
});

