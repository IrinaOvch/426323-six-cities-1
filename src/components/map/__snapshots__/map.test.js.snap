// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render map correctly 1`] = `
<section
  className="property__map map"
>
  <div
    id="map"
    style={
      Object {
        "height": "100%",
      }
    }
  />
</section>
`;
